main/src/main/java/com/sedmelluq/discord/lavaplayer/source/twitch/TwitchStreamAudioSourceManager.java

```diff
@@ -37,6 +37,7 @@
   private static final Pattern streamNameRegex = Pattern.compile(STREAM_NAME_REGEX);
 
   public static final String CLIENT_ID = "jzkbprff40iqj646a697cyrvl0zt2m6";
```

Comment:
Renaming this field to something like `DEFAULT_CLIENT_ID` might describe its new functionality more accurate